client/test/e2e/cash/cash.spec.js

```diff
@@ -202,7 +179,7 @@ describe('Cash Payments Module', function () {
       FC.click();
 
       // enter the amount to pay for a caution
-      components.currencyInput.set(mockCautionPayment.amount);
+      components.currencyInput.set(mockCautionPayment.amount, null);
```

Comment:
For future reference, you do not need to pass in `null` here.  It will default to `undefined`, which is false-y.